message_adapter/message_adapter.py

```diff
@@ -54,18 +53,17 @@ def load_and_update_remote_event(self, incoming_event, context):
         else:
             event = load_remote_event(event)
 
-        if context and 'meta' in event and 'workflow_tasks' in event['meta']:
-            cumulus_meta = event['cumulus_meta']
+        if context and 'meta' in event:
             task_meta = {}
             task_meta['name'] = context.get('function_name', context.get('functionName'))
             task_meta['version'] = context.get('function_version', context.get('functionVersion'))
             task_meta['arn'] = context.get('invoked_function_arn',
                                            context.get('invokedFunctionArn',
                                                        context.get('activityArn')))
-            task_name = get_current_sfn_task(cumulus_meta['state_machine'],
-                                             cumulus_meta['execution_name'],
-                                             task_meta['arn'])
-            event['meta']['workflow_tasks'][task_name] = task_meta
+            if not 'workflow_tasks' in event['meta']:
+                event['meta']['workflow_tasks'] = {}
+            task_index = len(event['meta']['workflow_tasks'])
```

Comment:
TIL you can do `len({})`

Reply:
I assume zero-indexed dict is fine:

```
>>> len({})
0
```

Don't see why it wouldn't be, just checking

Reply:
Yes, `len(dict()) == 0`.

Reply:
Yes, len(<dict>) returns the number of keys in python.   `len(dict()) == 0`